specification/peering/resource-manager/Microsoft.Peering/preview/2019-03-01-preview/peering.json

```diff
@@ -0,0 +1,1309 @@
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PeeringListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation has failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List peerings in a resource group": {
+            "$ref": "./examples/ListPeeringsByResourceGroup.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/peerings": {
+      "get": {
+        "tags": [
+          "Peerings"
+        ],
+        "description": "Lists all of the peerings under the given subscription.",
+        "operationId": "Peerings_ListBySubscription",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PeeringListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation has failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List peerings in a subscription": {
+            "$ref": "./examples/ListPeeringsBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PeeringListResult": {
+      "description": "The paginated list of peerings.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of peerings.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Peering"
+          }
+        },
+        "nextLink": {
+          "description": "The link to fetch the next page of peerings.",
+          "type": "string"
+        }
+      }
+    },
+    "Peering": {
+      "description": "Peering is a logical representation of a set of connections to the Microsoft Cloud Edge at a location.",
+      "required": [
+        "sku",
+        "kind",
+        "properties",
```

Comment:
>properties [](start = 9, length = 10)

Not sure whether we would like to mark properties as required !

Reply:
removed the required.